symplyphysics/laws/electricity/transmission_lines/wave_resistance_of_microstrip_line_for_effective_width_greater_thickness.py

```diff
@@ -0,0 +1,63 @@
+from sympy import Eq, solve, sqrt, log, pi
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless
+)
+
+## Description
+## The microstrip line is a dielectric substrate on which a metal strip is applied.
+## The wave resistance of a transmission line is a value determined by the ratio of the voltage of the incident
+## wave to the current of this wave in the transmission line.
+## When a wave propagates along a microstrip line, part of the field goes out, since the microstrip line does
+## not have metal borders on all sides, unlike, for example, rectangular waveguides. Then imagine an environment
+## in which the field will have the same magnitude as the field of a microstrip line. The permittivity of such a
+## medium will be called the effective permittivity of the line.
+## The effective width of a microstrip line is the width of such a flat capacitor, the electric intensity between the plates
+## of which is equal to the electric intensity in the dielectric of the substrate under the line strip.
+
+## Law is: Z = 120 * pi * (Wef / h + 1.393 + 0.667 * ln(Wef / h + 1.444))**(-1) / sqrt(ef), where
+## Z - wave resistance of the microstrip line,
+## Wef - effective width of the microstrip line,
+## h - thickness of substrate,
+## ef - effective permittivity of the microstrip line.
+
+# Conditions:
+# - the effective width of the microstrip line should be greater than the thickness of the substrate.
```

Comment:
Greater or equal?

Reply:
> Greater

Greater

Reply:
But you throw exception on less, not on less or equal.